packages/picasso.js/src/core/brush/brush.js

```diff
@@ -362,6 +392,48 @@ export default function brush({
    * @type {string}
    */
 
+
+  /**
+    * Configure the brush instance.
+    *
+    * @param {brush-config} config
+    * @example
+    * brushInstance.configure({
+    *   ranges: [
+    *     { key: 'some key', includeMax: false },
+    *     { includeMax: true, includeMin: true },
+    *   ]
+    * })
+    */
+  fn.configure = (config = {}) => {
+    if (Array.isArray(config.ranges) && config.ranges.length) {
+      rangeConfig = {
+        sources: {},
+        default: extend({}, DEFAULT_RANGE_CONFIG)
+      };
+
+      config.ranges.forEach((cfg) => {
+        const c = {};
+        Object.keys(DEFAULT_RANGE_CONFIG)
+          .filter(key => key !== 'key')
```

Comment:
should rename `key` to `attr` to be consistent with the next line and reduce confusion with `'key'`